ui/src/pages/help-center/HelpCenter.tsx

```diff
@@ -0,0 +1,131 @@
+import { CardsData, cardIds, startIcon } from './HelpCenter.constants';
+import { useUser } from 'contexts/user';
+
+export const HelpCenter: FC = () => {
+  const { user } = useUser();
+
+  const getIcon = useCallback((cardId: string): ReactNode => {
+    switch (cardId) {
+      case cardIds.pmmDocs:
+        return <KnowledgeBaseIcon />;
+      case cardIds.support:
+        return <Support />;
+      case cardIds.forum:
+        return <ForumOutlined />;
+      case cardIds.pmmDump:
+        return <DatasetOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const getButtonStartIcon = useCallback((iconName: string): ReactNode => {
+    switch (iconName) {
+      case startIcon.download:
+        return <SaveAlt />;
+      case startIcon.map:
+        return <MapOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const shouldDisplayCard = useCallback(
+    (itemId: string): boolean =>
+      !(
+        !user?.isPMMAdmin &&
+        (itemId === cardIds.pmmDump || itemId === cardIds.pmmLogs)
+      ),
+    [user]
+  );
+
+  return (
+    <Page title={Messages.pageTitle}>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: {
+            xs: '1fr',
+            sm: '1fr',
+            md: 'repeat(2, 1fr)',
+          },
+          gap: 4,
+        }}
+      >
+        {CardsData.map(
+          (item) =>
+            shouldDisplayCard(item.id) && (
+              <Card
+                style={{
+                  borderTop: item.borderColor
+                    ? `solid 12px ${item.borderColor}`
+                    : 'none',
+                }}
+                key={item.id}
+                data-testid={`help-card-${item.id}`}
+              >
+                <CardContent style={{ paddingRight: 16, paddingLeft: 16 }}>
+                  <div
+                    style={{
+                      display: 'flex',
+                      justifyContent: 'flex-start',
+                      alignItems: 'center',
+                      marginBottom: 8,
+                    }}
+                  >
+                    {getIcon(item.id)}
+                    <Typography
+                      variant="h6"
+                      style={{ marginLeft: getIcon(item.id) ? 8 : 0 }}
+                    >
+                      {item.title}
+                    </Typography>
+                  </div>
+
+                  <Typography>{item.description}</Typography>
+                  <div style={{ display: 'flex', paddingTop: 16 }}>
+                    {item.buttons.map((button) => (
+                      <Button
+                        key={button.url}
+                        variant="outlined"
+                        size="small"
+                        style={{ marginRight: 8 }}
+                        startIcon={getButtonStartIcon(button.startIconName)}
+                        endIcon={button.target && <NorthEast />}
+                        onClick={() => {
```

Comment:
Please extract into a separate function (have a `handleClick` method or something like that)

Also from an accessibility perspective it isn't good practice to use buttons as links (https://www.w3schools.com/accessibility/accessibility_buttons_links.php)

You can use the `component` prop on MUI components (for example `component={Link}` where Link is from react-router - used for navigating within the app) or the `href` attribute for external links (should render an `a` (anchor) on the page)